tests/python_dict/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -30,12 +44,26 @@ a = {
     "hhhhhhhhhhhhhh": 345
 }
 
+a = {}
+
+a = {}
+
+a = {
+    "short_key_1": 1,
+    "really_long_key_1_aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa":
+        "really_long_value_aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+    "short_key_2": 1,
+    "really_long_key_2_aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa":
+        "really_long_value_aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+}
+
 `;
 
 exports[`dict.py 2`] = `
 a = {
     "a": 123,
-    'b': 345
+    'b':
```

Comment:
why does this break on two lines?

Reply:
This is the part of the snapshot that corresponds to the input file, and I though it might be useful to see that it un-wraps things when they should be on a single line. If you look at [tests/python_dict/dict.py](https://github.com/prettier/plugin-python/pull/28/files/b6be4ca9a8a8f978e5f1ec0af469598b34b57714#diff-07fc735ca010ba1255a6ffaed1feff22) you'll see the same change :) If you disagree, I'm happy to revert this :D

Reply:
ah, right! sorry I got confused! thanks :)